eng/PrepareRelease.yml

```diff
@@ -66,8 +124,17 @@ stages:
           targetPath: '$(System.ArtifactsDirectory)\ReleaseStaging'
           publishLocation: 'pipeline'
           artifact: 'MonitorRelease'
-      - task: tagBuildOrRelease@0      
-        displayName: 'Tag Build with Monitor Release'
-        inputs:
-          type: 'Build'
-          tags: 'MonitorRelease'
+      - ${{ if 
+          and(
+            ne(variables['System.TeamProject'], 'public'), 
+            notin(variables['Build.Reason'], 'PullRequest'), 
+            or(
+              startsWith(variables['Build.SourceBranch'], 'refs/heads/release/'), 
+              startsWith(variables['Build.SourceBranch'], 'refs/heads/internal/release/')
+            )
+          ) }}:
+        - task: tagBuildOrRelease@0
+          displayName: 'Tag Build with Monitor Release'

```

Comment:
Is test/release for testing the release build? I'm assuming that's why you don't want to tag it?

Reply:
I took that part out, but yet, that was a specific branch we made to test the release pipeline, it needed to stay constant because we had to create a DARC channel for it.